Modules/Core/TestKernel/src/itkTestDriverIncludeRequiredFactories.cxx

```diff
@@ -41,12 +41,42 @@
 #include "itkOFFMeshIOFactory.h"
 #include "itkVTKPolyDataMeshIOFactory.h"
 
+// FFT
+#include "itkFFTImageFilterFactory.h"
+#include "itkVnlComplexToComplex1DFFTImageFilter.h"
+#include "itkVnlComplexToComplexFFTImageFilter.h"
+#include "itkVnlForward1DFFTImageFilter.h"
+#include "itkVnlForwardFFTImageFilter.h"
+#include "itkVnlHalfHermitianToRealInverseFFTImageFilter.h"
+#include "itkVnlInverse1DFFTImageFilter.h"
+#include "itkVnlInverseFFTImageFilter.h"
+#include "itkVnlRealToHalfHermitianForwardFFTImageFilter.h"
+
+#if defined(ITK_USE_FFTWD) || defined(ITK_USE_FFTWF)
+#  include "itkFFTWComplexToComplex1DFFTImageFilter.h"
+#  include "itkFFTWComplexToComplexFFTImageFilter.h"
+#  include "itkFFTWForward1DFFTImageFilter.h"
+#  include "itkFFTWForwardFFTImageFilter.h"
+#  include "itkFFTWHalfHermitianToRealInverseFFTImageFilter.h"
+#  include "itkFFTWInverse1DFFTImageFilter.h"
+#  include "itkFFTWInverseFFTImageFilter.h"
+#  include "itkFFTWRealToHalfHermitianForwardFFTImageFilter.h"
+#endif
```

Comment:
remove

Reply:
@thewtex Do you agree with above discussion that we should register FFTW factories by default to 1) align with behavior in consumer projects and 2) reduce test boilerplate?

Reply:
@tbirdso sorry, the discussion above was hidden, and I missed it. I followed up there. The goal here should be to provide a default, consistent FFT backend for tests. The behavior in consumer projects is should be different than unit tests, which are expected to have a consistent FFT backend. We can reduce test boilerplate in how the tests are written as much as possible, balancing readability and code coverage.

Reply:
Thanks for the explanation above @thewtex, I agree with the motive to "test one thing" for each test and see how swapping in FFTW classes via the object factory can violate this idea. I'll move forward with removing default FFTW registration in TestKernel. 

This might require revisiting FFT tests in a couple of places where it is assumed that the "best" backend has been registered first so as to have highest priority, and adding a dedicated test to verify that the object factory is functioning properly with Vnl/FFTW swappable backends. If the scope looks like it will expand beyond small fixes I will add an issue to track and bump to a separate PR.